### DISCUSSION TITLE
Legal Entity for Assets

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#55] [General]
While I do like this being a community effort and open source project, I do think that the long term sustainability and continuity likely requires some sort of legal entity that can own assets, handle money, and provide liability shielding for contributors.

I would propose a non-profit entity that does this, with the ability to hold funds used for various purposes, such as infrastructure. This would also allow Azure, AWS, etc,, to donate resources for hosting, email, etc.

The US has relatively few rules on a 501(3)(c) non proft - [https://en.wikipedia.org/wiki/501(c)(3)_organization](https://en.wikipedia.org/wiki/501(c)(3)_organization), though the US is also more litigious. 

I would also note that the US, with much of the vendors and sponsors located there, is perhaps easier to deal with for transferring funds.

Debate welcome.

My initial view would be that a board of directors (5-7) be elected somehow and hold 2-4 annual meetings for the purposes of strategic review of the group. I would also propose a group of 7-9 separate individuals act as a steering committee for ongoing operations. Each group would work for a term of 1-2 years.

[a comment by an automated account is not shown]

**Comment by amtwo** [comment]
If there is a legal entity, I think it _must_ be a proper 501(c)(3) or 501(c)(6) tax exempt organization. 

A 501(c)(3) includes organizations with educational missions. The main criteria to qualify would be for the primary purpose, and majority of spending be spent on educational programs. The theoretical organization would likely qualify for this, but we'd want to check with a tax attorney.

A 501(c)(6) is a business league -- like your local Chamber of Commerce, or Trade group. The NFL is a 501(c)(6). If the organization didn't qualify as a (c)(3), it would surely qualify as a (c)(6). 

The main difference is around tax deductions for payments _to_ the organization. Donations to a (c)(3) are pretty much always eligible as a tax-deductible donation, but payments to a (c)(6) are not deductible as donations--I believe they count as business expenses.


### All that said... Is it a good idea? 
**Probably**. I think it's dangerous for the stability of the community to have assets held by individuals. SQLBlog.com fell apart in part because a single individual held the domain & managed hosting...and those haven't been maintained. 

### When should it be formed?
I don't think we need to have it set up on Day 1--but I think when the project reaches "critical mass" we will need to take this step. By then, experience will also help clarify questions around qualifying for (c)(3) status, involved individuals, etc.

### What should it look like?
I have no idea. 

I think the "Board elected **by the membership**" concept of PASS was a failure. Despite the bureaucracy of the NomCom, elections were less about voting on the issues or qualified candidates, and were heavily just a popularity contest. This did not consistently produce a diverse Board of Directors, nor a Board that was skilled in the areas of need.

I think the Board should be selected by the Board itself, similar to [The Mozilla Foundation](https://wiki.mozilla.org/Board/FAQ#How_are_board_members_chosen.3F). The Board would then select the Steering Committee members. I would like to see decisions on structure based on research of how other successful non-profits (Tech/open-source, and otherwise) select their leadership. 

There should be a dedicated role as part of Board member or Steering Committee member(s) that is dedicated to Diversity, Inclusion, Culture, and Belonging--- a "Culture & Inclusion Chair" or something. In fact--this should probably happen even without a legal entity, but particularly with one. Having Inclusion run _throughout_ every aspect of any organization or project is going to be critical to success. It is a constant problem in Tech, and requires enough effort, research, knowledge, and experience that having it "assigned" to someone to help lead efforts.

**Reply by rabryst** [reply]
I served on the final Nominations Committee and found it to be an interesting experience, and not that bad as an concept. In practice it operated "above the law", meaning that it wanted to do what was best for the community — if the NomCom wanted to make a change to policy, it was enabled to do so (for example, 2021 would have had clarity on what constituted acceptable behaviour by candidates). Long conversations were had and the membership (under an excellent leadership of course) was thoughtful. In particular I liked the need for absolute agreement in order to approve or decline a candidate.

Of course the election itself is just a popularity contest, and in my opinion it wasn't long enough, and engagement was clearly very low as a percentage of membership.

[NDA stuff here which I'm thinking but not typing.]

With all that said, we'll eventually need a business that can accept money and own assets, but it's not necessary *right* now, but we do need a plan. Rob says elsewhere that he'd prefer not having a US company and I agree, but I also believe that having a US org will help with sponsorships. Perhaps we can follow some charity models and incorporate in Europe and US to start. That's a tax nightmare but it might be worth pursuing down the line.

[a comment by an automated account is not shown]

**Comment by way0utwest** [comment]
Good thoughts. I've been hesitant to have self-selected individuals have too much of a say, but maybe that's not a bad choice. 

I need to do this for T-SQL Tuesday as well, either in a legal entity here or another one.

**Reply by rabryst** [reply]
For a new org self-selection is fine, as long as there's a plan in place to replace yourself after the first term or two. I have some thoughts about NomCom which I'll put as a reply to Andy's comments.

[a comment by an automated account is not shown]

**Comment by amtwo** [comment]
I wouldn't close the door to having a mechanism for the Board (or part of the Board) to be seated in a way other than self-selecting. But that begs the question of "by whom?" ... It just can't be "everyone." 

It would really need to be determined based on targeted constituencies, and I don't think should replace board-appointed board seats. Perhaps there would a seat elected by/for Speakers, and a seat elected by/for Organizers, and a seat elected by/for GitHub contributors.... But we need to identify those constituencies, and we would need some sort of track record to be able to form them. So even in an initial form, all seats would likely need to be appointed

[a comment by an automated account is not shown]

**Comment by gavincampbell** [comment]
I'd be wary of incorporation unless the central org _needed_ to handle money - and if the central services can be provided without spending anything then this can be avoided for a long time - consulting that tax attorney might cost as much as the whole of Data Saturday Pordenone, and our recent experience suggests that HQ organisations are very good at generating costs for themselves which then need to be met somehow.

Agree there needs to be some mechanism for deciding who gets to decide which events get listed - these people will probably be the maintainers of the website. AFAIK the Python Steering council is unincorporated. Python also has a non-profit foundation, separate from the steering council, which is currently asking the public for donations following the cancellation of its conference!

Final note on this is that DevOpsDays has a _for-profit_ partner called ConferenceOps, which does billing and accounts etc. for those events that aren't set up to do it themselves, but it's optional for events to use them and it certainly didn't exist on day 1. I think it might be US only.

[a comment by an automated account is not shown]

**Comment by gavincampbell** [comment]
Oh, regarding voting. This used to be my field, I could bore for England on the subject. https://en.wikipedia.org/wiki/Schulze_method seems to be popular with the kids.

[a comment by an automated account is not shown]

**Comment by SQLDBAWithABeard** [comment]
My tuppence

I would personally choose to avoid a US based organisation for two reasons

- The litigiousness 
- Alienation of the rest of the world

Until there is a need to handle money or assets, I am unsure as to the value proposition for an organisation.

There is an option which suggests that we could gain sponsorship for any required assets - Domain Names, infrastructure are the only two I see without requiring an organisation to hold any funds.

That said, I am not averse to some organisation within this open-source, community collaborative for decision making but right now, ensuring an organic growth and use of the resources is more important in my opinion.

The community resources that are already growing, from DCAC, from Microsoft, from community members are already replacing some of the facilities that were previously provided and it is worth holding out for a short period of time and letting the dust settle and see where this ends up.

[a comment by an automated account is not shown]

**Comment by gavincampbell** [comment]
If the infrastructure is just GitHub pages, then this doesn't need any sponsorship. Email is a bit more problematic, but can be done for free up to a certain size. As far as I can see the only asset is the domain name.

[a comment by an automated account is not shown]

**Comment by SQLDBAWithABeard** [comment]
Would Discussions be a better place for this conversation?

I have set up discussions on the repo and the link is here

https://github.com/sqlcollaborative/DataSaturdays/discussions

also available on the front page

**Reply by amtwo** [reply]
I took the "convert issue to discussion" button for a spin, and it seems to work great! 😁

[a comment by an automated account is not shown]

**Comment by way0utwest** [comment]
I would add a couple points here.
1. Part of the reason for growth, and the sustenance of many smaller events was the global sponsorship. Many vendors want to be able to provide a pool of money, but not manage that with individual organizers. There is a need for this, whether through this effort or another.
2. Establishing a non profit is not expensive or hard. Many user groups have done this, without any issues or big costs. 
3. Azure (or other) credits for hosting are available for non profits. Not sure how this works for OSS projects, but hosting could be a nontrivial, though not necessarily large, expense. (https://www.microsoft.com/en-us/nonprofits/azure)

I am not proposing a US entity, but as I pointed out, many sponsors are US based. I might lean towards Norway, as they have the history with the Nobel organization and promoting worldwide collaboration and peace, so if this does not interfere with any banking issues,that is a consideration.

**Reply by lowlydba** [reply]
Great note on the free stuff angle. When I've worked for non-profits, the amount of free or heavily discounted software was immense. Aside from cloud credits, a lot of other tools/services could be opened up by having non profit status in the US.

**Reply by amtwo** [reply]
I view PASS's failure to achieve IRS recognition as a 501(c) tax free entity to be a big miss for that org, exactly for the "free stuff" angle.

**Reply by gavincampbell** [reply]
Wasn't one of the complaints about PASS that the global sponsorship all got spent on HQ costs and there was very little "trickle down" to the non-summit events?

**Reply by amtwo** [reply]
Yes. The [FY2020 budget for PASS](https://user-images.githubusercontent.com/9381581/103062811-6143e480-457d-11eb-8d71-a06edbc45abe.png) was about $8MM, but included less than $250k in expenses for user groups (local groups + virtual) & SQLSat events. (Note that "Virtual Events" in the budget refers to HQ-led virtual events.) 3% of the budget is spent on community-led events (UGs & SQLSats), while nearly 24% is spent on "Corporate Administration".

I think the (perceived) mismanagement of PASS resources is off-topic, and a separate issue from the fact that there was a legal entity that backed the organization. In fact, I'd argue that if there isn't a legal entity to handle any funds & expenses, then mismanagement is harder to identify.

I think it best to keep this conversation focused on the questions of:
* Should there be a legal entity?
* If there is a legal entity, what should it look like?

I feel strongly PASS is not the correct model to use when thinking about what the legal entity should look like, so my last comment is probably a bit off-topic, too. :) I think keeping ourselves focused on where we're going will keep this discussion thread moving forward to a decision point. Apologies for starting to veer the conversation down a tangent about that other org.

**Reply by gavincampbell** [reply]
My point was more that global sponsorship has never really been a thing for most event organisers - only 20% of that 3% was SQLSaturday sponsorship, the rest was overheads - so providing a structure to enable global sponsorship may not seem like a priority.

**Reply by eugman** [reply]
> My point was more that global sponsorship has never really been a thing for most event organisers - only 20% of that 3% was SQLSaturday sponsorship, the rest was overheads - so providing a structure to enable global sponsorship may not seem like a priority.

Thanks for clarifying your intent, that makes it a lot clearer and is a good point. 👍

**Reply by way0utwest** [reply]
I'm not sure what you point is here with the global sponsorship. For most organizers, the money was welcome and used, as they had less other sponsorships to raise. Especially for smaller events, of which there are many.

The model PASS used did take a lot of that money for overhead. I don't know if it was over 50% (I think it was under that), but it was way more than necessary. Supporting global sponsorships, and making this easy for vendors doesn't need to cost much for any other entity. Using PASS as a model for how to move forward, or how to create a profit for someone, isn't the intent here.

**Comment by MarkDataGuy** [comment]
Hi.

If you plan to seek global sponsorship, then you need some sort of legal entity. In an Irish context, a club would be the simplest structure, allowing you have a bank account and governed by club constitution, however officers are legally accountable for club action and at least two will need to be signatures on club bank account. The club structure assumes everyone is a volunteer. 

After club, you are looking at either a charity or limited company, both come with basic overhead costs associated with maintaining the entity (accounts filing etc), but offer protection and is a "proper " legal entity to o own assets, employ people, etc.

While I agree that most global sponsors are likely to be US based, these sponsors will have international offices so the actual location of the legal datasaturday should not be an issue. Typically they will need you to become an official vendor (if you are a looking for a large amount of money as I assume you are thinking) so that vendor on boarding will handle the payment setup irrespective of country of the organisation. 

I would think for now, just run the organisation as a "virtual club" to make decisions. Have a "trusted" group of initial decision makers, but that same group of decision makers would then have to outline the long term structure and governance. This of course has to be transparent, and not have any unremovable executive authority. You probably should have a voted board, but that board has to have at executive authority. They need to be able to carry out their decisions, assuming they have been properly voted on according to organisation  rules. Who votes is interesting. If your remit is to run datasaturdays, the perhaps each datasaturday gets a vote, subject to some time limit. Maybe combine that with club "trustees" votes which would have to be a balanced (and appropriately diversified) group of community members, which may include sponsor representatives, speaker representatives and attendee representatives. 

That's my random 2 cents. Best of luck.

Mark

**Reply by MarkDataGuy** [reply]
Just to clarify, the suggestion of data Saturday organisers and community representatives having a vote is to have a vote on the who is on the board. The board takes it's own vote on matter arising to running of the organisation.  Membership of board and changes to organisation constitution is voted on by the wider organisation membership.

**Reply by amtwo** [reply]
I'm not bought in on the idea of having broad membership vote on board membership (see previous comments [here](https://github.com/sqlcollaborative/DataSaturdays/discussions/55#discussioncomment-237956) and [here](https://github.com/sqlcollaborative/DataSaturdays/discussions/55#discussioncomment-237960)). With PASS that turned into a popularity contest. It was not an effective way to bring change to the organization, and ultimately, I think the elected board was detrimental to the organization.

**Reply by MarkDataGuy** [reply]
I didn't suggest that board membership vote on board membership. Some things are constrained around your legal organisation. I can only speak around Irish structure as that is what I understand. If you for a club, you must have members, a constitution and a board that is voted on by the members. That is how a club runs. If you have a limited company, the you much have share holders, a company secretary, and at least one board meeting a year, the share holders have the executive authority.  You dont need any structure if you prefer, but then the executive authority is owned by whoever set it up. 

If you want to create an organisation that is a legal entity, the you choose the legal entity and it defines how things are governed. 

As a personal opinion, all votes are a popularity contest, you just hope the popular choice is also competent. 

Mark

**Comment by way0utwest** [comment]
OK, I am continuing this discussion, since I think this is a better time. The asset sale is settled, so we have a direction.

I would start by saying that I think this project, DataSaturdays.com, ought not to build a legal entity for tools and the project. I  am not sure it is needed, and there is a level of independence for the technical side of things v the running of events. 

That being said, my proposal to Redgate (disclosure, I work for Redgate) is that they give the SQL Saturday franchise to a non profit to hold in perpetuity. Redgate would be a foudning member, and perhaps there would be other founding members. Some of the founding members would have a seat on the board, but I would aim to structure this so that the majority of the board cannot be overruled by founding members. I don't quite know how to structure this, so ideas welcome.

I would see this entity hosting the SQL Saturday site (whatever that is), and potentially offering to host DataSaturdays.com as well. There is some discussion here [SSC](https://www.sqlservercentral.com/forums/topic/data-saturdays-and-sqlsaturday) and here {113](https://github.com/sqlcollaborative/DataSaturdays/discussions/113)  about how to structure the brands together, but I would like some merging.

I see this legal entity as doing the following:
- hosting the main attendee site for various domains
- providing a non profit entity to allow vendors to donate to events (already had some request for this from organizers in various geographies)
- providing a non profit entity for global partner programs (donate $xxx for all events, split among them)
- provide a hosting platform for events that need this (free or very, very low $)
- provide broadcast facilities for events (free or low $)
- provide a succession plan for a board and committees that allows this to outlive dependencies on any one person
- require no salaries for volunteers
- operate on < US$5k/yr
- publish transparent books on board meetings and financials as they change.

For me, I'd like to drop tsqltuesday.com and sqlmemorial.org under something like this as well.

I do think a legal entity smooths some items out, allows work to move forward, and helps many organizers out that do not want to run their own entity.
